Assets/VRTK/Scripts/Interactions/VRTK_InteractTouch.cs

```diff
@@ -64,6 +66,8 @@ public class VRTK_InteractTouch : MonoBehaviour
         private VRTK_ControllerEvents.ButtonAlias originalGrabAlias;
         private VRTK_ControllerEvents.ButtonAlias originalUseAlias;
 
+        public GameObject ControllerAlias { get { return controllerEvents.ControllerAlias; } }
```

Comment:
Yeah its used by `InteractGrab` which I forgot to commit. Though I've added it now. Good pointing out that even the internal implementation might as well use the property though.